roles/k8s_best_practices_certsuite/tasks/tests.yml

```diff
@@ -33,8 +33,8 @@
   block:
     - name: Run the Red Hat Best Practices Test Suite for Kubernetes test cases
       vars:
-        kbpc_partner_repo: "{{ (kbpc_registry | length) | ternary(kbpc_registry, 'quay.io') }}/redhat-best-practices-for-k8s"
-        kbpc_support_image: "{{ kbpc_support_image_name }}:{{ kbpc_support_image_version }}"
```

Comment:
This line must be present and cannot be deleted: `kbpc_support_image: "{{ kbpc_support_image_name }}:{{ kbpc_support_image_version }}"`

Reply:
```suggestion
        kbpc_support_image: "{{ kbpc_support_image_name }}:{{ kbpc_support_image_version }}"
```